docs-en/chapter_data_structure/number_encoding.md

```diff
@@ -2,11 +2,11 @@
 
 !!! note
 
-    In this book, chapters marked with an * symbol are optional reads. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
+    In this book, chapters marked with an * symbol are optional to read. If you are short on time or find them challenging, you may skip these initially and return to them after completing the essential chapters.
 
 ## Integer Encoding
 
-In the table from the previous section, we noticed that all integer types can represent one more negative number than positive numbers, such as the `byte` range of $[-128, 127]$. This phenomenon, somewhat counterintuitive, is rooted in the concepts of sign-magnitude, one's complement, and two's complement encoding.
+In the table from the previous section, we noticed that all integer types can represent one more negative number than positive numbers, such as the `byte` range of $[-128, 127]$. This phenomenon seems counterintuitive, and its underlying reason involves knowledge of sign-magnitude, one's complement, and two's complement encoding.
```

Comment:
"In the table from the previous section, we noticed that all integer types can represent one more negative number than positive numbers, such as the `byte` range of $[-128, 127]$. This phenomenon seems counterintuitive, and its underlying reason involves knowledge of sign-magnitude, one's complement, and two's complement encoding."
can be changed to ->
"In the table presented in the preceding section, we observed that that all integer types can represent one more negative number than positive numbers, for instance, the 'byte' type has a range of [-128, 127]. This phenomenon might seem counterintuitive, and its root cause lies in the knowledge of sign-magnitude, one's complement, and two's complement encoding."

Reply:
Thank you so much. I revisited the meaning and usage of the words “observe” and “notice”. "Observe" is more suitable in this context.